docker-compose.yml

```diff
@@ -23,6 +23,10 @@ services:
   #
   #             https://selenium-python.readthedocs.io/api.html#module-selenium.webdriver.common.proxy
   #
+  #       Playwright proxy settings playwright_proxy_server, playwright_proxy_bypass, playwright_proxy_username, playwright_proxy_password
```

Comment:
Does playwright run in its own container instead? or generally not? could be better for keeping the changedetection.io image size low? also can be a problem for the raspberry-pi/arm people who use this platform

Reply:
Yeah this is my main problem now with this PR, it also needs to run on Arm devices, or be decoupled somehow so it doesnt atleast break ARM compilation.. do you know something about that situation?

Reply:
It installs Chromium inside the main container. There is a [chromium package for arm64 Bullseye](https://packages.debian.org/bullseye/arm64/chromium) but I currently doesn't have any raspberry-pi/arm to test it. Maybe:

- build this as another tag e.g. 0.39.10-playwright *or*
- create a [browser server](https://playwright.dev/docs/api/class-browsertype#browser-type-launch-server) (maybe something like [this](https://github.com/karatelabs/karate/tree/master/karate-core#playwright)) container and connect to it using [p.chromium.connect()](https://playwright.dev/python/docs/api/class-browsertype#browser-type-connect) instead of `p.chromium.launch()`?

Reply:
so one thing then please, can you go with adding the container to the`docker-compose.yml` (commented out initially), and have the changedetection.io app use its won env variable called `PLAYWRIGHT_DRIVER_URL"

like with 

https://github.com/dgtlmoon/changedetection.io/blob/9fc291fb631bc4120ee8c304270911c0b61d9fa0/changedetectionio/content_fetcher.py#L72

Reply:
OK, done.